src/controller_examples/zookeeper_controller/exec/reconciler.rs

```diff
@@ -24,12 +24,17 @@ pub struct ZookeeperReconcileState {
     // reconcile_step, like a program counter, is used to track the progress of reconcile_core
     // since reconcile_core is frequently "trapped" into the controller_runtime spec.
     pub reconcile_step: ZookeeperReconcileStep,
+    pub old_sts: Option<StatefulSet>,
```

Comment:
The name `old_sts` is a bit weird because there is no new one here. Maybe name it as `sts_from_get`